Automattic/wp-calypso
# Performance Profiler: Remove unused components removed in #95108

Related to https://github.com/Automattic/wp-calypso/pull/95108

## Proposed Changes

A follow up to https://github.com/Automattic/wp-calypso/pull/95108 which removes the V1 components in favour of the V2 components. This PR removes the now unused components and CSS.

Note that that `<SiteProfilerV2>` component still exists because it's still possible to access the old component depending on the feature flags.

Refactors like this can be hard to read, so I've broken the PR up into more readable commits. This is the main one: https://github.com/Automattic/wp-calypso/commit/6782fd6a937d84ae12a03d1b874996d84d2ac594

Changes:
* Delete old `CoreWebVitalsAccordion`, `CoreWebVitalsDetails`, and `MetricTabBar` components and rename their `V2` variants
* Tidy CSS class names
* Remove the `<MetricScale>` component which is no longer referenced
* Fix a typo in the `metricsThresholds` utility

## Why are these changes being made?


It's better to tidy up dead CSS sooner rather than later.

## Testing Instructions



There should be no changes, a pure refactor.

* Test logged in view at `http://calypso.localhost:3000/sites/performance/philipmjackson76.wpcomstaging.com?filter=all`
* Test logged out view at `http://calypso.localhost:3000/speed-test-tool?url=https://philipmjackson76.wpcomstaging.com`

Compare with Figma Iix1FFHIQdfECIXn5JKT5s-fi-5426_141260

## Screenshots

**Desktop logged in**
![desktop logged in Sites _ My WordPress Site _ WordPress_com](https://github.com/user-attachments/assets/afea9c55-25a2-43a6-91a5-e7a9e7f1ed0a)

**Mobile logged in**
![mobile logged in Sites _ My WordPress Site _ WordPress_com](https://github.com/user-attachments/assets/7d39dbdd-bbba-4f0d-b58b-d73c13986ee1)


**Desktop logged out**
![desktop logged out Speed Test _ WordPress_com](https://github.com/user-attachments/assets/d9d0b74e-92a8-4b2e-b2e4-ac2e53aa06ca)

**Mobile logged out**
![mobile logged out Speed Test _ WordPress_com](https://github.com/user-attachments/assets/04c5ecc2-a389-42e8-b75b-afe0a015753d)

## Pre-merge Checklist
